app/services/expression_evaluators/measure_unit.rb

```diff
@@ -36,7 +36,7 @@ def measure_unit_answers
     end
 
     def measure_applicable_units
-      commodity.applicable_measure_units.select do |_unit, values|
+      filtered_commodity.applicable_measure_units.select do |_unit, values|
```

Comment:
I don't know where this came from. A rebase with master was done, but that should not add a change against it.

Should this be removed from this PR?

Reply:
Yes please